README.md

```diff
@@ -9,306 +9,25 @@
-sudo apt install python3.10-venv
-python -m venv .env
-source .env/bin/activate
-
-# Setup MaxText.
-cd maxtext/
-bash setup.sh
-```
-
-## Step 3: Convert Model Checkpoints 
-
-You can run the JetStream MaxText Server with Gemma and Llama2 models. This section describes how to run the JetStream MaxText server with various sizes of these models.
-
-### Use a Gemma model checkpoint
-
-*   You can download a [Gemma checkpoint from Kaggle](https://www.kaggle.com/models/google/gemma/frameworks/maxText/variations/7b). 
-*   After downloading checkpoints, copy them to your GCS bucket at `$CHKPT_BUCKET`.
-    *   `gsutil -m cp -r ${YOUR_CKPT_PATH} ${CHKPT_BUCKET}`
-    *   Please refer to the [conversion script](https://github.com/google/JetStream/blob/main/jetstream/tools/maxtext/model_ckpt_conversion.sh) for an example of `$CHKPT_BUCKET`.
-*   Then, using the following command to convert the Gemma checkpoint into a MaxText compatible unscanned checkpoint.
-
-```bash
-# bash ../JetStream/jetstream/tools/maxtext/model_ckpt_conversion.sh ${MODEL} ${MODEL_VARIATION} ${CHKPT_BUCKET}
-
-# For gemma-7b
-bash ../JetStream/jetstream/tools/maxtext/model_ckpt_conversion.sh gemma 7b ${CHKPT_BUCKET}
-```
-
-Note: For more information about the Gemma model and checkpoints, see [About Gemma](https://github.com/google/maxtext/blob/main/end_to_end/gemma/Run_Gemma.md).
-
-
-### Use a Llama2 model checkpoint
-
-*   You can use a Llama2 checkpoint you have generated or one from [the open source community](https://llama.meta.com/llama-downloads/). 
-*   After downloading checkpoints, copy them to your GCS bucket at `$CHKPT_BUCKET`.
-    *   `gsutil -m cp -r ${YOUR_CKPT_PATH} ${CHKPT_BUCKET}`
-    *   Please refer to the [conversion script](https://github.com/google/JetStream/blob/main/jetstream/tools/maxtext/model_ckpt_conversion.sh) for an example of `$CHKPT_BUCKET`.
-*   Then, using the following command to convert the Llama2 checkpoint into a MaxText compatible unscanned checkpoint.
-
-```bash
-# bash ../JetStream/jetstream/tools/maxtext/model_ckpt_conversion.sh ${MODEL} ${MODEL_VARIATION} ${CHKPT_BUCKET}
-
-# For llama2-7b
-bash ../JetStream/jetstream/tools/maxtext/model_ckpt_conversion.sh llama2 7b ${CHKPT_BUCKET}
-
-# For llama2-13b
-bash ../JetStream/jetstream/tools/maxtext/model_ckpt_conversion.sh llama2 13b ${CHKPT_BUCKET}
-```
-
-Note: For more information about the Llama2 model and checkpoints, see [About Llama2](https://github.com/google/maxtext/blob/main/getting_started/Run_Llama2.md).
-
-
-## Step4: Run the JetStream MaxText server
-
-
-### Create model config environment variables for server flags
-
-You can export the following environment variables based on the model you used.
-
-*   You can copy and export the `UNSCANNED_CKPT_PATH` from the model\_ckpt\_conversion.sh output.
-
-
-#### Create Gemma-7b environment variables for server flags
-
+## JetStream Engine Implementation 
 
+There are two engine implementation - Jax and Pytorch.
```

Comment:
How about "Currently, there are two reference engine implementations available -- one for Jax models and another for Pytorch models."

Reply:
Thanks, changed.